sdk/python/tests/test_bigquery_ingestion.py

```diff
@@ -16,65 +17,75 @@
 
 
 @pytest.mark.integration
-def test_bigquery_ingestion_correctness():
-    # create dataset
-    ts = pd.Timestamp.now(tz="UTC").round("ms")
-    data = {
-        "id": [1, 2, 1],
-        "value": [0.1, 0.2, 0.3],
-        "ts_1": [ts - timedelta(minutes=2), ts, ts],
-        "created_ts": [ts, ts, ts],
-    }
-    df = pd.DataFrame.from_dict(data)
+class TestBigQueryIngestion:
```

Comment:
Just a naming nitpick, but should this be called something like `TestMaterializeFromBigQueryToLocal`? `TestBigQueryIngestion` makes it seem like we're testing a ingestion API for batch data into BQ.